web/src/tippyjs.ts

```diff
@@ -426,7 +426,7 @@ export function initialize(): void {
     });
 
     delegate("body", {
-        target: "#email_invite_radio_container.disabled_setting_tooltip",
+        target: "#invite_users_option_tabs_container.no_email_invites",
```

Comment:
I think you probably want the tooltips to have as its target the actual tab that's disabled; the current look, where you get the tooltip hovering over the current tab, and it's centered above the boundary between the tabs, looks a bit odd.

Reply:
Working on it.